cursorless-talon/src/cursorless.talon

```diff
@@ -25,6 +25,6 @@ cursorless docks:          user.cursorless_open_instructions()
 cursorless reference:      user.cursorless_cheat_sheet_show_html()
 cursorless cheat sheet:    user.cursorless_cheat_sheet_show_html()
 cursorless settings:
-    user.vscode("workbench.action.openGlobalSettings")
+    user.fs_run_command("workbench.action.openGlobalSettings")
```

Comment:
This only works in VSCode.  This also does beg the question if we want to create some higher-level actions or something so that we can start to abstract the editor away a bit more.

Reply:
Not sure if you want me to do anything here.

Reply:
I'd follow the approach above and make this an action which is implemented in VSCode context. The code here is quite VSCode-specific

Reply:
Done, created a new Python file in actions as that looked like the pattern but should we move these global actions into on file?